blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueTable.java

```diff
@@ -0,0 +1,32 @@
+package io.jenkins.blueocean.rest.model;
+
+import com.google.common.annotations.Beta;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A Table-like data structure for serving tabular data
+ */
+@ExportedBean
+@Beta
```

Comment:
Beta? I guess `Beta` annotation will go away once you are happy with this API? Not entirely sure what purpose it solves. We should avoid using Guava in model.

Reply:
Removed in 7b25272